Airports/EGLL/Ownership.txt

```diff
@@ -70,7 +70,7 @@ BORDER:LLTWR
 DEPAPT:EGLL
 
 SECTOR:WUAPP:0:2000
-OWNER:WUA:LLN:LLS:LLF:TCNW:TCN:TC:LC:LNC:LWC:LSC:L
+OWNER:WUA:LLN:LLS:LLF:TCNW:TCN:TC:LCE:LC:LNC:LWC:LSC:L
```

Comment:
Appreciate this isn't really related to this issue - but our vMATS defines the 'Heathrow Group' as including Northolt, and covered top-down by TC SE. Am wondering what makes the most sense...?

Reply:
agreed I have rehomed this with TCSE